src/main/java/pong/PongFrame.java

```diff
@@ -1,4 +1,6 @@
 package pong;
 
 public class PongFrame {
```

Comment:
I add these final variables into the various classes in order to be able to determine the correct starting position of the ball or if the ball collides with the paddle.